### ISSUE TITLE
Cetus pro V1.1 multiple firmware troubles

### ISSUE THREAD

**Issue by tano68** [BETAFPV/BETAFPV_Configurator#13]
Hello,
Hard time here. After many mistakes done navigating through all the variation of the configurator (V3 & co), I'm with a firmware blank Cetus. 
What's working : 
-Read parameters and configure with configurator V1.0.0 and the older version
-Load separate firmware whit V1.1.0
-Erase firmware whit V3
What's not working :
-Load OSD and Sensor firmware in Configurator V1.1.0
-Connect to Configurator V1.1.0, even after updating the main firmware
-Can't read the drone parameters a second time with configurator V1.0.0

Long story short, my cetus is blinking frenetically red lack of firmware.
I sincerly tried everything found here and elsewhere and nothing solved my issue.
Thanks

**Comment by tano68** [comment]
I forget also, sensor and OSD firmware aren’t recognised by Configurator V1.1.0 (the main reason I can’t load them)

**Comment by HWT662988** [comment]
Cetus pro V1.1 should use the firmware which named 1.1.x ,because the 1.2.x is for the  Cetus pro V1.2 board .
![image](https://user-images.githubusercontent.com/45007731/151134342-3428e8a7-3555-4727-a696-dcc0740dc121.png)

**Comment by tano68** [comment]
That’s what I’ve done. Your answer is off topic. 
my problems are :
1 no OSD and sensor firmwares in my Cetus
2 can’t load them because the ones available aren’t recognised by configurator
3 each different versions of configurator allow me to do different task which aren’t available in an other.

**Comment by wenzhicode** [comment]
Hi, tano68.

> 

which version firmware did you try to flash and where you download them ?

**Comment by wenzhicode** [comment]
> That’s what I’ve done. Your answer is off topic. my problems are : 1 no OSD and sensor firmwares in my Cetus 2 can’t load them because the ones available aren’t recognised by configurator 3 each different versions of configurator allow me to do different task which aren’t available in an other.

So I guess it maybe be not load the correct firmware to the board. Can you take a picture of FC board so I can try what you done in that board? Thanks.

**Comment by wenzhicode** [comment]
> Hello, Hard time here. After many mistakes done navigating through all the variation of the configurator (V3 & co), I'm with a firmware blank Cetus. What's working : -Read parameters and configure with configurator V1.0.0 and the older version -Load separate firmware whit V1.1.0 -Erase firmware whit V3 What's not working : -Load OSD and Sensor firmware in Configurator V1.1.0 -Connect to Configurator V1.1.0, even after updating the main firmware -Can't read the drone parameters a second time with configurator V1.0.0
> 
> Long story short, my cetus is blinking frenetically red lack of firmware. I sincerly tried everything found here and elsewhere and nothing solved my issue. Thanks

Thanks for your trying, I can't agree more that this process is a hard time. You can also contract our sale-supporter and get a refresh if you don't want a another hard try.

**Comment by FPVWILLY** [comment]
I have the same problem but with the 2.0 FC
